react/features/analytics/middleware.js

```diff
@@ -12,9 +12,12 @@ import { initAnalytics } from './functions';
  */
 MiddlewareRegistry.register(store => next => action => {
     switch (action.type) {
-    case LIB_DID_INIT: {
+    case SET_ROOM: {
```

Comment:
SET_ROOM will also happen when we navigate to the welcome page, and the room will be `undefined`. Do we also want to do this in that case?

Reply:
We want to have send analytics for the welcome page and I don't think we can set a room. I think undefined is fine.

Reply:
👍